src/drivers/cloudflare-kv-http.ts

```diff
@@ -96,13 +96,21 @@ export default defineDriver<KVHTTPOptions>((opts) => {
   const kvFetch = $fetch.create({ baseURL, headers })
 
   const hasItem = async (key: string) => {
-    const { success } = await kvFetch(`/values/${key}`)
-    return success
+    try {
+      await kvFetch(`/values/${key}`)
+      return true
+    } catch {
+      return false
+    }
   }
 
   const getItem = async (key: string) => {
-    const { result } = await kvFetch(`/values/${key}`)
-    return result
+    try {
+      // Cloudflare API returns with `content-type: application/octet-stream`
+      return await kvFetch(`/values/${key}`, { parseResponse: txt => txt })
+    } catch {
+      return null
```

Comment:
I think we should only assume 404 is valid response and not hide any other (token or network errors)